examples/config/grid_config.config

```diff
@@ -0,0 +1,36 @@
+<?xml version="1.0"?>
+
+<plugin filename="MinimalScene">
+    <ignition-gui>
+      <title>View 1</title>
+      <property type="string" key="state">docked</property>
+    </ignition-gui>
+    <engine>ogre2</engine>
+    <scene>scene</scene>
+    <ambient_light>1 1 1</ambient_light>
+    <background_color>0.8 0.8 0.8</background_color>
+    <camera_pose>-6 3 6 0 0.5 0</camera_pose>
+</plugin>
+<plugin filename="InteractiveViewControl" name="Interactive view control">
+  <ignition-gui>
+    <anchors target="View 1">
+      <line own="right" target="right"/>
+      <line own="top" target="top"/>
+    </anchors>
+    <property key="resizable" type="bool">false</property>
+    <property key="width" type="double">5</property>
+    <property key="height" type="double">5</property>
+    <property key="state" type="string">floating</property>
+    <property key="showTitleBar" type="bool">false</property>
+  </ignition-gui>
+</plugin>
+<plugin filename="GridConfig" name="Grid config">
```

Comment:
Can we document the child elements (e.g., `cell_count`, `vertical_cell_count`) somewhere more visible?

Reply:
Good call, documented on the header.